.github/workflows/deploy_api.yml

```diff
@@ -48,15 +48,15 @@ env:
   OPENBLAS_NUM_THREADS: 1
 
   # Optional Snapshot number to use pyseir model output from. An empty string by default
-  PYSEIR_ARTIFACT_SNAPSHOT: ${{ github.event.inputs.pyseir_snapshot }}  
+  PYSEIR_ARTIFACT_SNAPSHOT: ${{ github.event.inputs.pyseir_snapshot }}
 
 jobs:
   build-and-publish-snapshot:
     runs-on: self-hosted
     steps:
     - name: Parse covid data model branch name and set env variable
       run: |
-        echo "COVID_DATA_MODEL_REF=${GITHUB_REF##*/}" >> $GITHUB_ENV
+        echo "COVID_DATA_MODEL_REF=${GITHUB_REF_NAME}" >> $GITHUB_ENV
```

Comment:
GITHUB_REF is something like "refs/heads/<branch_name>" and so we were stripping off everything up to the (last) `/` character to get the branch name, but that made branches like `mikelehen/foo` not work correctly.  But it looks like we can just use `GITHUB_REF_NAME` which will be something like `mikelehen/foo`.  

https://docs.github.com/en/actions/learn-github-actions/environment-variables